locales/en/kafkaoverview.json

```diff
@@ -1,14 +1,27 @@
 {
   "contactSalesCardCallToActionButton": "Contact sales",
-  "contactSalesCardMainText": "Contact sales to get a pre-paid subscription or modify limits of your current subscription. Pre-paid subscriptions are available to customers in any location, including locations outside the United States and Canada.",
+  "contactSalesCardMainText": "Contact sales to get a pre-paid subscription or modify limits of your current subscription.",
   "contactSalesCardTitle": "Contact sales",
   "heroCallToActionButton": "",
   "heroDescription": "Red Hat OpenShift Streams for Apache Kafka is a managed cloud service that provides a streamlined developer experience for building, deploying, and scaling real-time applications in hybrid-cloud environments. The combination of seamless operations across distributed microservices, large data transfer volumes, and managed operations allows teams to focus on core competencies, accelerate application velocity and reduce operational cost.",
   "heroTagline": "Fully hosted and managed service for stream-based applications",
   "heroTitle": "Get started with Red Hat OpenShift Streams for Apache Kafka",
   "heroTryItButton": "",
   "purchaseCardCallToActionButton": "Purchase now",
-  "purchaseCardMainText": "Use Red Hat Marketplace to purchase a pay-as-you-go subscription for up to 25 streaming units. Purchasing through Red Hat Marketplace is only available in the United States and Canada.",
+  "purchaseCardMainText": "In the United States or Canada, purchase a pay-as-you-go subscription for up to 25 streaming units through Red Hat Marketplace.",
   "purchaseCardTitle": "Purchase now",
-  "viewDocumentation": ""
+  "viewDocumentation": "",
+  "pricingModalTitle": "Pricing model",
+  "steamingUnit": "Streaming unit",
+  "dataTransfer": "Data transfer",
+  "storage": "Storage",
+  "kafkaInstanceCapacityTitle": "Kafka instance capacity",
+  "kafkaInstanceCapacityDescription": "OpenShift Streams offers instances in the following sizes with the following capacity.",
+  "kafkaInstanceCapacityfooter": "*Storage can be extended. If you wish to increase the storage limit for a Kafka instance, contact <support>Red Hat Support</support> to discuss your options. You can check your current storage usage by <value>monitoring disk space metrics</value>.",
+  "cloudProvidersTitle": "Cloud providers",
+  "aswDescription": "Available now",
```

Comment:
While trying to finalize the design, I also received feedback that this label should not be included. Instead, we should just show the "Coming soon" text for Azure.